garnett/fields.py

```diff
@@ -75,6 +76,11 @@ def validate_translation_dict(all_ts):
             raise exceptions.ValidationError(f'Invalid value for language "{code}"')
 
 
+def translatable_default(inner_default):
```

Comment:
Typing??

Reply:
I didn't add it because there is no type checker running in the ci and no typing elsewhere. Is probably good to add it for documentation alone though